src/main/java/reposense/parser/CsvParser.java

```diff
@@ -62,8 +62,8 @@ public CsvParser(Path csvFilePath) throws IOException {
             }
         } catch (IOException ioe) {
             throw new IOException(MESSAGE_UNABLE_TO_READ_CSV_FILE, ioe);
-        } catch (ParseException pe) {
-            logger.log(Level.WARNING, pe.getMessage(), pe);
+        } catch (ParseException | IllegalArgumentException e) {
+            logger.log(Level.WARNING, e.getMessage(), e);
```

Comment:
We can have a separate catch block and logging message for both exceptions, also easier for us to trace which error occur.

Reply:
Not sure if I understood this correctly, they currently already have different messages, i.e MESSAGE_ILLEGAL_FORMATS and INVALID_COMMIT_HASH_MESSAGE.